### ISSUE TITLE
Source map references file not in npm package

### ISSUE THREAD

**Issue by qidydl** [eBay/nice-modal-react#108]
The just-released version 1.2.9 added source mapping, but the source map references a file that isn't included in the published package. This results in an error like the following:

```
WARNING in ./node_modules/@ebay/nice-modal-react/lib/esm/index.js
Module Warning (fom ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '<path>\node_modules\@ebay\nice-modal-react\src\index.tsx' file: Error: ENOENT: no such file or directory, open '<path>\node_modules\@ebay\nice-modal-react\src\index.tsx'
```

(I had to retype this message by hand for unrelated reasons so apologies if I typoed anything)

This doesn't appear to actually break anything, so it's probably a low-priority error, but the source map likely doesn't work.

**Comment by szubitsky** [comment]
I just did an upgrade to the new version and can confirm the same error message. However, I haven't noticed it breaking anything.

**Comment by kbzowski** [comment]
+1

**Comment by supnate** [comment]
Resolved in the latest version`1.2.10`.
